docs/AboutUs.md

```diff
@@ -7,5 +7,5 @@ Display | Name | Github Profile | Portfolio
 ![](https://via.placeholder.com/100.png?text=Photo) | John Doe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Liu Haoran | [Github](https://github.com/yuqiaoluolong) | [Portfolio](docs/team/yuqiaoluolong.md)
```

Comment:
Perhaps you might need to pull in the new merge and update this file. When docs/team/filename.md is used the website cannot be published. Therefore consider changing it to team/filename.md.

Reply:
OK

Reply:
I deleted the changes and may create a new PR later for that.